roundware/rw/models.py

```diff
@@ -73,6 +73,7 @@ class Project(models.Model):
     speak_enabled = models.BooleanField(default=False)
     geo_speak_enabled = models.BooleanField(default=False)
     reset_tag_defaults_on_startup = models.BooleanField(default=False)
+    timed_asset_priority = models.BooleanField(default=False)
```

Comment:
The default is currently `True`, since timed assets have priority. This reverses that. IMO Timed assets should generally be the default, but IDK how you are using this feature. In any case, to make this "right" we need either:

1. default=True, the way it works now.
2. A data migration to change all existing Project objects to have timed_asset_priority=True since we don't want functionality surprises on existing installs.